src/server/worker/worker_service.proto

```diff
@@ -30,6 +31,13 @@ message CancelResponse {
 service Worker {
   rpc Status(google.protobuf.Empty) returns (pps.WorkerStatus) {}
   rpc Cancel(CancelRequest) returns (CancelResponse) {}
+  rpc CollectChunk(CollectChunkRequest) returns (stream google.protobuf.BytesValue) {}
```

Comment:
To match with our other API calls could we call this `GetChunk`? `Collect` isn't a keyword we use anywhere else and this seems to basically be the same behavior as our other `Get` calls.

Reply:
Sure.